media-tv/kodi/kodi-9999.ebuild

```diff
@@ -133,6 +133,11 @@ DEPEND="${COMMON_DEPEND}
 # generated addons package.  #488118
 [[ ${PV} == "9999" ]] && DEPEND+=" virtual/jre"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-9999-no-arm-flags.patch #400618887
+	"${FILESDIR}"/${PN}-9999-texturepacker.patch
+)
```

Comment:
@rindeal while that is obviously true in general, `PATCHES` has a well defined, global meaning in `EAPI=6`, intended for containing patches. This is not some for loop counter, hence it can stay global as it is.